bridges/BakaUpdatesMangaReleasesBridge.php

```diff
@@ -0,0 +1,84 @@
+<?php
+class BakaUpdatesMangaReleasesBridge extends BridgeAbstract {
+	const NAME = 'Baka Updates Manga Releases';
+	const URI = 'https://www.mangaupdates.com/';
+	const DESCRIPTION = 'Get the latest series releases';
+	const MAINTAINER = 'fulmeek';
+	const PARAMETERS = array(array(
+		'series_id' => array(
+			'name'			=> 'Series ID',
+			'exampleValue'	=> '12345'
+		)
+	));
+	const LIMIT_ITEMS = 10;
+
+	private $feedName = '';
+
+	public function collectData() {
+		$series_id = preg_replace('/[^0-9]/', '', $this->getInput('series_id'));
+		if (empty($series_id))
+			returnServerError('Invalid series ID');
+
+		$html = getSimpleHTMLDOM(self::URI . 'releases.html?search=' . $series_id . '&stype=series')
```

Comment:
Suggestion: Override the function `getURI` to return the URL for the current request. That way it gets included with the feed (i.e. feed readers will open the correct URL instead of the home page).

https://github.com/RSS-Bridge/rss-bridge/wiki/The-getURI-function